velox/expression/tests/ExpressionVerifierUnitTest.cpp

```diff
@@ -102,7 +102,8 @@ TEST_F(ExpressionVerifierUnitTest, persistReproInfo) {
     auto plan = parseExpression("always_throws(c0)", asRowType(data->type()));
 
     removeDirecrtoryIfExist(localFs, reproPath);
-    VELOX_ASSERT_THROW(verifier.verify({plan}, data, nullptr, false), "");
+    VELOX_ASSERT_THROW(
+        verifier.verify({plan}, data, std::nullopt, nullptr, false), "");
```

Comment:
Could you also add a test case of verify() applying only on a subset of rows? (E.g., maybe make the unselected rows throw if they are evaluated, then assert verify() doesn't throw.)